### DISCUSSION TITLE
Render Svelte Components within the Markdown content?

### DISCUSSION THREAD

**Discussion by bwklein** [ssssota/svelte-exmarkdown#13] [Ideas]
One of the benefits of MDsveX is that you can use Svelte Components within the Markdown and they will render correctly as part of the parent Markdown component.  Is there a way to do this with your renderer?

**Comment by ssssota** [comment]
Thank you for asking!

Unfortunately, svelte-exmarkdown does not allow Svelte components to be written within Markdown.

However, you can draw any component by creating and using plug-ins. For example, mermaid for drawing diagrams and katex for drawing formulas.
 
The svelte-exmarkdown documentation uses these plug-ins for code highlighting and drawing diagrams.
See the [`/src/prism`](https://github.com/ssssota/svelte-exmarkdown/tree/main/src/prism), and [`/src/mermaid`](https://github.com/ssssota/svelte-exmarkdown/tree/main/src/mermaid) directories.
